addons/sale_project/models/sale_order.py

```diff
@@ -41,6 +41,7 @@ def _compute_project_ids(self):
             projects = order.order_line.mapped('product_id.project_id')
             projects |= order.order_line.mapped('project_id')
             projects |= order.project_id
+            projects = projects.filtered(lambda p: p.active)
```

Comment:
Thanks for your work. 🙂 

After discussing with Alexandra (AJU) and Yannick (YTI), we concluded to display all projects in the action instead of filtering the projects and project count. Because we think it should be better for the user to give him all the data to allow him to quickly see the tasks done one of his archived projects linked instead of completely losing that possibility. 🙂 

Can you do the changes accordingly?

Thanks in advance! 🙂